src/commands/junit/README.md

```diff
@@ -23,17 +23,19 @@ datadog-ci junit upload --service my-service --tags key1:value1 --tags key2:valu
 ```
 
 - The positional arguments are the directories or file paths in which the jUnit XML reports are located. If you pass a folder, the CLI will look for all `.xml` files in it.
+- `--apiKey` (default: `DATADOG_API_KEY` env var) is the API Key used to authenticate the requests. **Important**: the API key needs to be available either through this parameter or `DATADOG_API_KEY` env var.
```

Comment:
```suggestion
- `--api-key` (default: `DATADOG_API_KEY` env var) is the API Key used to authenticate the requests. **Important**: the API key needs to be available either through this parameter or `DATADOG_API_KEY` env var.
```

Reply:
the problem is that other commands use `apiKey`: https://github.com/DataDog/datadog-ci/blob/master/src/commands/synthetics/run-test.ts#L338. It's probably better to keep consistency among commands @AdrianLC

Reply:
I'm not so sure if the other commands are not even consistent themselves. Anyway it's not important whatever you think it's best

Reply:
we're going to remove `--api-key` so that there's less risk when dealing with secrets, so this is irrelevant